studyzen/Courses/Requests/CreateFlashCardRequest.cs

```diff
@@ -0,0 +1,19 @@
+using System.Text.Json.Serialization;
+using StudyZen.Common;
+
+namespace StudyZen.FlashCards.Requests;
+
+public sealed class CreateFlashCardRequest
+{
+ 
+    public int FlashCardSetId {get;}
+    public string Question { get; }
+    public string Answer { get; }
+
+    public CreateFlashCardRequest(int flashCardSetId, string? question, string? answer)
+    {
+        FlashCardSetId = flashCardSetId;
+        Question = question.ThrowIfRequestArgumentNull(nameof(question));
```

Comment:
we shouldn't throw exception as user could want to modify only the question or only the answer